src/routers/handlers/individual-psc-list/individualPscListHandler.ts

```diff
@@ -161,7 +161,7 @@ export class IndividualPscListHandler extends GenericHandler<IndividualPscListVi
                 .filter(name => name)
                 .join(" "); // ensure single space between names even if some are missing
             const idvDetails = psc.identityVerificationDetails;
-            const requestExtensionUrl = companyNumber ? addSearchParams(ExternalUrls.REQUEST_AN_EXTENSION, { companyNumber, selectedPscId, lang }) : undefined;
+            const requestExtensionUrl = companyNumber ? addSearchParams(env.PSC_EXTENSIONS_PATH, { companyNumber, selectedPscId, lang }) : undefined;
```

Comment:
Perhaps it would be clearer to keep ExternalUrls.REQUEST_AN_EXTENSION here and redefine its value in constants.ts